result/isislsp.yaml

```diff
@@ -0,0 +1,390 @@
+          type: boolean
+        attached_delay:
+          description: |-
+            Delay Metric - when set, the originator is attached to another
+            area using the referred metric.
+          type: boolean
+        attached_default:
+          description: |-
+            Default Metric - when set, the originator is attached to another
+            area using the referred metric.
+          type: boolean
+        overload:
+          description: |-
+            Overload bit - when set, the originator is overloaded, and must
+            be avoided in path calculation.
+          type: boolean
+
+    IsisLsp.IsReachability:
+      description: >-
+        This container describes list of ISIS neighbors and attributes in IS-Reachability TLV 2.
+      type: object
+      properties:
+        neighbors: 
+          description: >-
+            This container describes Intermediate System (IS) neighbors.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.neighbor'
+
+    IsisLsp.ExtendedIsReachability:
+      description: >-
+        This is list of ISIS neighbors and attributes in Extended-IS-Reachability TLV 22.
+      type: object
+      properties:
+        neighbors: 
+          description: >-
+            This container describes IS neighbors.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.neighbor'
+
+    IsisLsp.neighbor:
+      description: >-
+        This contains IS neighbors.
+      type: object
+      properties:
+        system_id:
+          description: >-
+            The System ID for this emulated ISIS router, e.g. "640100010000".
+          type: string
+          format: hex
+
+    IsisLsp.Ipv4InternalReachability:
+      description: |-
+        This container defines list of IPv4 internal reachability information.
+        This is advertised when the origin-type is set 'internal' in route range configurations.
+      type: object
+      properties:
+        prefixes:
+          description: >-
+            Describes list of IPv4 prefixes in this TLV.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.V4Prefix'
+
+    IsisLsp.Ipv4ExternalReachability:
+      description: |-
+        This container defines list of IPv4 external reachability information.
+        This is advertised when the origin-type is set 'external' in route range configurations.
+      type: object
+      properties:
+        prefixes:
+          description: >-
+            Describes list of IPv4 prefixes in this TLV..
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.V4Prefix'
+
+    IsisLsp.V4Prefix:
+      description: >-
+        This group defines attributes of an IPv4 standard prefix.
+      type: object
+      properties:
+        ipv4_address:
+          description: >-
+            An IPv4 address.
+          type: string
+        prefix_length:
```

Comment:
If we agree on min-max as being necessary for LI fields, then prefix lengths for v4/v6 should have min max of 0/32 and 0/128 respectively,